content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImportWaitForVm.class/__methods__/wait_for_imported_vm.rb

```diff
@@ -1,36 +1,42 @@
+#
+# Description: This method checks if the VM has been successfully imported
+#
+
 module ManageIQ
   module Automate
-    module System
-      module Event
-        module EmsEvent
-          module RHEVM
-            class UpdateVmImportStatus
```

Comment:
@smelamud What is the event triggered by ovirt when a vm is created during v2v? For example during vm creating process we have two events (we have more but for sake of discussion I simplify ) `USER_ADD_VM` and `USER_ADD_VM_FINISHED_SUCCESS`. The last event triggers refresh to update the inventory. I wonder why do we need it at the beginning?

BTW do we know what is the difference between two v2v runs when a vm is inventory before and after the process completes? It sounds like a bug to me.